app/main.py

```diff
@@ -1,8 +1,46 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: int, clean_power: int,
+                 average_rating: int, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> int:
+        clean_difference = self.clean_power - car.clean_mark
+        price = car.comfort_class * clean_difference * self.average_rating
+        price /= self.distance_from_city_center
+        return round(price, 1)
+
+    def wash_single_car(self, car: Car) -> bool:
```

Comment:
No need to use return here, u can provide calculations in serve_cars method

Reply:
I don`t understand. Readme says, that function should return calculation. And code example has return in this function.